openscap_daemon/cli_helpers.py

```diff
@@ -263,8 +263,8 @@ def summarize_standard_compliance_results(arf_source, result_list):
 
     test_result = arf_root.find(
         ".//cdf:TestResult[@id='%s']" %
-        ("xccdf_org.open-scap_testresult_xccdf_org.ssgproject.content_profile_"
-         "standard"), namespaces
+        # this ID prefix is hardcoded in oscap
+        ("xccdf_org.open-scap_testresult_" + profile), namespaces
```

Comment:
@mpreisler I don't think so. This prefix is part of TestResult@id of every XCCDF results that is created by OpenSCAP. I think it isn't SSG specific. If some content that isn't from SSG is evaluated, the XCCDF results contain this prefix also. Actually, what was here before was SSG specific. 

You can find the string in https://github.com/OpenSCAP/openscap/blob/maint-1.2/src/XCCDF_POLICY/xccdf_policy.c#L2011

Anyway, it's just a comment, I wrote it there to explain the "magic constant", but there's no problem to remove the comment.